src/mrpro/operators/_FourierOp.py

```diff
@@ -59,13 +59,17 @@ def __init__(
         super().__init__()
 
         def get_spatial_dims(spatial_dims, dims):
-            return [s for s, i in zip((spatial_dims.z, spatial_dims.y, spatial_dims.x), (-3, -2, -1)) if i in dims]
+            return [
+                s
+                for s, i in zip((spatial_dims.z, spatial_dims.y, spatial_dims.x), (-3, -2, -1), strict=False)
+                if i in dims
+            ]
 
         def get_traj(traj, dims):
-            return [k for k, i in zip((traj.kz, traj.ky, traj.kx), (-3, -2, -1)) if i in dims]
+            return [k for k, i in zip((traj.kz, traj.ky, traj.kx), (-3, -2, -1), strict=False) if i in dims]
 
         self._ignore_dims, self._fft_dims, self._nufft_dims = [], [], []
-        for dim, type in zip((-3, -2, -1), traj.type_along_kzyx):
+        for dim, type in zip((-3, -2, -1), traj.type_along_kzyx, strict=False):
```

Comment:
why strict=False?

either strict=True or disabling the check....

Reply:
Yeah, saw this just after I pushed the commit. It was an auto adjustment by ruff. Will change to true.

Reply:
changed in [c0e2d68](https://github.com/PTB-MR/mrpro/pull/193/commits/c0e2d68acdd11289bc3819520efef2bef172fa2c)